Javascript(ES5)/README.md

```diff
@@ -1193,7 +1734,9 @@ Forked from the excellent [Airbnb JavaScript Style Guide](https://github.com/air
     this.firstName = 'Panda';
     ```
 
-  - Don't save references to this. Use Function#bind.
+  - Don't save references to this. Use [`Function#bind`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind).
```

Comment:
IMO, nothing. The current guides that we have set a preference for using `something.bind(this)()`, so that internally you can use `this` without saving references from somewhere else.

I am not used to that, but I kind of catch the drift, and I like it.

Do you think we should remove this recommendation?